### PR TITLE
chore: add optional inputs for github deployments

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-4029]

### Description

We have specific development environments that we want to keep active.
https://github.com/chrnorm/deployment-action

### How to test

- check https://github.com/toptal/picasso/pull/3589

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

[FX-4029]: https://toptal-core.atlassian.net/browse/FX-4029?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: create-jira-deployment/action.yml [hunk 1]
```diff
@@ -12,6 +12,12 @@ inputs:
   environment:
     required: true
     description: 'Name for the target deployment environment'
+  transient-environment:
+    required: false
+    description: 'Specifies if the given environment is specific to the deployment and will no longer exist at some point in the future.'
+  auto-inactive:
+    required: false
+    description: Adds a new inactive status to all prior non-transient, non-production environment deployments with the same repository and environment name as the created status's deployment. An inactive status is only added to deployments that had a success state. Default true
```
Comment (dmaklygin) [inline create-jira-deployment/action.yml:20]: let's add a default value here as well?

**Review by sashuk — APPROVED** [review]
Should work as advertised in https://github.com/chrnorm/deployment-action/tree/main#action-inputs

File: create-jira-deployment/action.yml (hunk 1 above)
Comment (TomasSlama) [reply create-jira-deployment/action.yml]: ✅ done

File: create-jira-deployment/README.md [hunk 2]
```diff
@@ -10,11 +10,13 @@ Installs package dependencies. Caches `node_modules` for faster subsequent insta
...
+| `environment`           | string | ✅        |         | Name for the target deployment environment                                                                                                                                                                                                                             |
+| `transient-environment` | string |          |         | Specifies if the given environment is specific to the deployment and will no longer exist at some point in the future.                                                                                                                                                 |
+| `auto-inactive`         | string |          |         | Adds a new inactive status to all prior non-transient, non-production environment deployments with the same repository and environment name as the created status's deployment. An inactive status is only added to deployments that had a success state. Default true |
```
[the remark this answers is not in the document]
Comment (TomasSlama) [reply create-jira-deployment/README.md]: ✅ done

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

File: create-jira-deployment/action.yml [hunk 3]
```diff
@@ -12,6 +12,14 @@ inputs:
...
     required: true
     description: 'Name for the target deployment environment'
+  transient-environment:
+    required: false
+    description: 'Specifies if the given environment is specific to the deployment and will no longer exist at some point in the future.'
+    default: 'true'
+  auto-inactive:
+    required: false
+    description: Adds a new inactive status to all prior non-transient, non-production environment deployments with the same repository and environment name as the created status's deployment. An inactive status is only added to deployments that had a success state.
+    default: 'true'
```
Comment (dmaklygin) [inline create-jira-deployment/action.yml:22]: hm, shouldn't it be a just raw boolean?

Comment (TomasSlama) [reply create-jira-deployment/action.yml:22]: it throws eslint error, I was checking how it is done in rest of the actions and when boolean, it is done like this

**Review by dmaklygin — APPROVED** [review]
Good job!

**Pull request merged**